src/data-scraper/index.ts

```diff
@@ -7,23 +7,59 @@ import { IRuntimeImagesResponse } from '../transmitter/types';
 import { NeedleOptions } from 'needle';
 import { Agent as HttpsAgent } from 'https';
 
+const httpsAgent = new HttpsAgent({
+  keepAlive: true,
+  // We agreed with Sysdig to skip TLS certificates validation for HTTPS connection.
+  rejectUnauthorized: false,
+});
+
+function getSysdigUrl(): string {
+  const base: string = config.SYSDIG_ENDPOINT;
+  const url: string = base + '/v1/runtimeimages';
+  return url;
+}
+
+function getSysdigAuthHeader(): string {
+  return `Bearer ${config.SYSDIG_TOKEN}`;
+}
+
 function isSuccessStatusCode(statusCode: number | undefined): boolean {
   return statusCode !== undefined && statusCode >= 200 && statusCode < 300;
 }
 
+/** NOTE: This function can throw, so the caller should handle errors. */
```

Comment:
nit: do we have to throw?

Reply:
Not necessary, but I would like the healthcheck to log if we detect an exception (the throwing is not only because of us, but also it could be the HTTP client returning a network error like ECONNRESET)